ERROR2023-07-12 09:15:29,746D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:165 JsonPath值获取失败 $['agent_name'] 
ERROR2023-07-12 09:19:49,460D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:165 JsonPath值获取失败 $['agent_name'] 
ERROR2023-07-12 11:02:37,994D:\Agent_Api\logs\error-2023-07-12.log:mysql_control.py:142 数据库连接失败，失败原因 tuple index out of range
ERROR2023-07-12 11:15:31,795D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:166 JsonPath值获取失败 $.agent_name 
ERROR2023-07-12 13:49:45,112D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:166 JsonPath值获取失败 $.agentName 
ERROR2023-07-12 13:50:26,156D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:166 JsonPath值获取失败 $.agent_name 
ERROR2023-07-12 13:55:20,328D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:166 JsonPath值获取失败 $.data.agent_name 
ERROR2023-07-12 13:56:36,894D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:166 JsonPath值获取失败 $.data.agent_name 
ERROR2023-07-12 13:58:36,649D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:166 JsonPath值获取失败 $.data.agent_name 
ERROR2023-07-12 13:59:22,169D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:166 JsonPath值获取失败 $.data.agent_name 
ERROR2023-07-12 14:55:57,871D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:170 JsonPath值获取失败 $.data.agent_name 
ERROR2023-07-12 15:30:24,341D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:168 JsonPath值获取失败 $.sql_data.agent_name 
ERROR2023-07-12 15:30:39,820D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:168 JsonPath值获取失败 sql_data.agent_name 
ERROR2023-07-12 17:02:44,844D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:168 JsonPath值获取失败 $['agent_name'] 
ERROR2023-07-12 17:19:05,573D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:168 JsonPath值获取失败 $.agent_name 
ERROR2023-07-12 17:19:32,179D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:168 JsonPath值获取失败 $.sql_data.agent_name 
ERROR2023-07-12 18:47:46,914D:\Agent_Api\logs\error-2023-07-12.log:assertion_control.py:168 JsonPath值获取失败 $.agentName 
